### PR TITLE
[APP]: add readme file

### PR DESCRIPTION
**Pull request by Someboty**
First version of README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+## Roles:
+
+In our API, functionality is divided for Customers and Managers:
+
+| Controller               | Customer                                                                                            | Manager                              |
+|--------------------------|-----------------------------------------------------------------------------------------------------|--------------------------------------|
+| AuthenticationController | Registration, authentication                                                                        | -                                    |
+| CarController            | Get list of cars, get car by id                                                                     | Create, update, delete functionality |
+| PaymentController        | Create a new payment, get all payments by user ID                                                   | -                                    |
```
Comment (ConfusedSatlan) [inline README.md:89]: Manager can get all payments of users

**Review by ConfusedSatlan — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+<h3>Welcome to the CarSharingApp API project, where we bring you an exceptional online car rental experience!</h3>
+Our robust and secure application, built on the **Java Spring Boot framework**, is designed to cater to all car enthusiasts. We've harnessed cutting-edge technologies and integrated a range of features to offer users a safe, all-encompassing, and user-friendly e-commerce platform. In the following sections, we will walk you through the essential aspects and impressive functionalities of our application.
+
+## Table of Contents
+
+- [👨‍💻Tech stack](#tech-stack)
+- [⚡Quick start](#-quick-start)
+- [✈️Features](#features)
+- [Roles](#roles)
```
Comment (Denis-Balako) [inline README.md:15]: add emoji

File: README.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+git clone https://github.com/jv-jun23-team1/car-sharing-app.git
+```
+
+Then we should build a project using **Maven**:
+```bash
+mvn clean install
+```
+Then, rise a **Docker** container of your app:
+```bash
+docker build -t {your-image-name}
```
Comment (Denis-Balako) [inline README.md:57]:
```suggestion
docker build -t {your-image-name} .
```

**Review by Denis-Balako — COMMENTED** [review]
Left some suggestions

File: README.md (hunk 2 above)
Comment (Someboty) [reply README.md:15]: Where did you get them?

**Review by keda-vlad — APPROVED** [review]

**Review by Denis-Balako — APPROVED** [review]

**Pull request merged**